net.c

```diff
@@ -2152,7 +2153,7 @@ int _ifcfg_write(char *device, ifcfg_t *ifcfg)
 
   // set hostname, if requested
   if(config.net.sethostname) {
```

Comment:
Good point. I think that it *somehow* works, because this was the only place in the source that mentions [`DHCLIENT_SET_HOSTNAME`](https://github.com/openSUSE/linuxrc/search?utf8=%E2%9C%93&q=DHCLIENT_SET_HOSTNAME) ... ... ... and the answer is: [lxrc_init](https://github.com/openSUSE/linuxrc/blob/3b0c3621093304fd5319ca3edb4f4338a94b3156/linuxrc.c#L780)